mithril-aggregator/src/event_store/database/migration.rs

```diff
@@ -20,5 +20,17 @@ create table if not exists event (
 );
             "#,
         ),
+        SqlMigration::new(
+            2,
+            r#"
+create view if not exists metrics_per_day as select date(metric_date) as date, action as counter_name, sum(counter) value from 
```

Comment:
We can change a little bit the view to compute the date in source query and create an index on this extracted date from json.
```
create view if not exists metrics_per_day as select metric_date as date, action as counter_name, sum(counter) value from 
    (
        select action, json_extract(content, '$.content.counter') counter, date(json_extract(content, '$.content.date')) metric_date 
        from event 
        where source='Metrics'
    ) 
group by action, date;

create index metric_date_index on event(date(json_extract(content, '$.content.date')));
```
For 1'000'000 of event, when I search metrics for a specific date, it takes 0.324s instead of 2.312s without this index.

I propose to add this index

Reply:
Note that `date` keep only the day of the date.

Reply:
OK :tada: That sounds like very decent performances.

Reply:
Done